src/tracks/playabletrack/wavetrack/ui/CutlineHandle.cpp

```diff
@@ -160,32 +140,7 @@ UIHandle::Result CutlineHandle::Click
    // Cutline data changed on either branch, so refresh the track display.
    UIHandle::Result result = RefreshCell;
 
-   if (event.LeftDown())
-   {
-      if (mLocation.typ == WaveTrackLocation::locationCutLine)
-      {
-         mOperation = Expand;
-         mStartTime = viewInfo.selectedRegion.t0();
-         mEndTime = viewInfo.selectedRegion.t1();
-
-         // When user presses left button on cut line, expand the line again
-         double cutlineStart = 0, cutlineEnd = 0;
-         mpTrack->ExpandCutLine(mLocation.pos, &cutlineStart, &cutlineEnd);
-         viewInfo.selectedRegion.setTimes(cutlineStart, cutlineEnd);
-      }
-      else if (mLocation.typ == WaveTrackLocation::locationMergePoint) {
```

Comment:
This eliminates the left click action to expand cut lines, but that was not in the scope of the issue.

Reply:
Ouch - thanks.